contracts/Core/AssetManager.sol

```diff
@@ -88,7 +64,7 @@ contract AssetManager is ACL, AssetStorage, Constants, StateManager, IAssetManag
         string calldata selector,
         string calldata url
     ) external onlyRole(ASSET_MODIFIER_ROLE) notState(State.Commit, parameters.epochLength()) {
-        require(jobs[jobID].assetType == uint8(AssetTypes.Job), "Job ID not present");
+        require(jobs[jobID].id == jobID, "Job ID not present");
```

Comment:
What if jobID==0?

Reply:
should require jobID!=0 to avoid unexpected issues in future.

Reply:
added https://github.com/razor-network/contracts/pull/468/commits/a0d2e3a74aa2d7ed80617dddc3c475fe9603bb62